src/PKSim.Presentation/Presenters/Parameters/ParametersByGroupPresenter.cs

```diff
@@ -51,7 +51,6 @@ public class ParametersByGroupPresenter : ParameterSetPresenter<IParametersByGro
          _representationInfoRepository = representationInfoRepository;
          _contextMenuFactory = contextMenuFactory;
          ShowFavorites = false;
-         HeaderVisible = false;
```

Comment:
Also ensuring that process parameter header is visible (does not look as nice but his consistent with rest of UI)